clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/content/PendingIntentFactory.kt

```diff
@@ -304,18 +309,16 @@ internal object PendingIntentFactory {
                 launchIntent!!.putExtra(PTConstants.PT_INPUT_AUTO_OPEN, renderer?.pt_input_auto_open)
                 launchIntent!!.putExtra("config", renderer?.config)
 
-                return if (renderer?.deepLinkList != null) {
-                    setPendingIntent(
-                        context,
-                        notificationId,
-                        extras,
-                        launchIntent,
-                        requestCode
-                    )
-                } else {
+                if (renderer.deepLinkList == null) {
```

Comment:
why did we invert the condition from != to ==

Reply:
The logic inside the if condition has been inversed as well. Earlier it was a little confusing and had redundant code.